mock/chairs/starting_a_group.md

```diff
@@ -0,0 +1,23 @@
+# Starting a Group
+## Menu
+* [Is starting a group the right course of action?]() (below)
+* [Incubation and WICG](#) (below)
+* [Community Groups](#) (below)
```

Comment:
Business and Interest Groups, for requirements?

Reply:
Added these sections